frontend/src/v5/ui/routes/dashboard/projects/federations/federationSettingsForm/federationSettingsForm.component.tsx

```diff
@@ -209,14 +209,13 @@ export const FederationSettingsForm = ({ open, federation, onClose }: IFederatio
 				name="defaultView"
 				label={formatMessage({ id: 'federations.settings.form.defaultView', defaultMessage: 'Default View' })}
 			/>
-			<SectionTitle>GIS servey point</SectionTitle>
+			<SectionTitle>GIS survey point</SectionTitle>
```

Comment:
Didn't notice this before. It should be localisable:

```suggestion
			<SectionTitle> 
				<FormattedMessage
					id="federations.settings.form.gisTitle"
					defaultMessage="GIS survey point"
				/>
			</SectionTitle>
```

Reply:
Fixed